packages/ansi-parse/ansi-parse.0.3.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+license: "ISC"
+maintainer: "Joel Jakubovic"
+authors: "Joel Jakubovic"
+dev-repo: "https://github.com/jdjakub/ansi-parse.git"
+homepage: "https://github.com/jdjakub/ansiparse"
+bug-reports: "https://github.com/jdjakub/ansiparse/issues"
```

Comment:
`homepage` and `bug-reports` are wrong here. The repository is named `ansi-parse`.